.travis.yml

```diff
@@ -1,41 +1,52 @@
 sudo: required
 dist: trusty
-addons:
-  firefox: latest
-  apt:
-    sources:
-    - google-chrome
-    packages:
-    - google-chrome-stable
-  sauce_connect: true
+language: node_js
+
 env:
   global:
   - secure: IbrybaUaJDiwNh8VodFfTIAdSw2Nl4DGRh2oDIO2vL8SPgGilwxY0go5DWrxAG0aP+YIXFkK+SpblFppQqqxnAoLVYmd7kFSJoOZraNqmcJqUYSzrHrWArmuiGzY8gMgOWTsnbzVVNRldN8tpoSwpQB/+YhCtbYb9eqvw+u5Izk=
   - secure: grvHX6c0ReQwRsuQZferI+uyGsKlubujURKuJB8coNdA9lgZcvf+nmCkNBpSML83hSjQkJUTt8XnucNNgL/1VxUhgy0O/wkTUEmcDAU1b80As8NJkbreJcc3HFASA68wou4HoJsFmJV2cpBYoAJJSX+HyT6mYd6pBh09tPfOeBg=
-  matrix:
-  - TEST_COMMAND="npm run test:unit"
-  - TEST_COMMAND="npm run test:integration" SKIP_REMOTE_BROWSERS=true
-  - TEST_COMMAND="npm run test:integration" RUN_SAUCE_TESTS=true
-matrix:
-  exclude:
-  - env: TEST_COMMAND="npm run test:integration" RUN_SAUCE_TESTS=true
-    node_js: '8'
-  - env: TEST_COMMAND="npm run test:integration" SKIP_REMOTE_BROWSERS=true
-    node_js: '9'
-language: node_js
-node_js:
-- '8'
-- '9' # TODO(https://github.com/Polymer/tools/issues/280): update to node 10
```

Comment:
Maybe add this command just above the matrix so that we are tracking it still?

Reply:
Done.